snap/info_snap_yaml.go

```diff
@@ -178,6 +180,7 @@ func InfoFromSnapYaml(yamlData []byte) (*Info, error) {
 			SocketMode:      yApp.SocketMode,
 			ListenStream:    yApp.ListenStream,
 			BusName:         yApp.BusName,
+			Environment:     yApp.Environment,
```

Comment:
it's a bit strange (and makes the reader wonder) doing a copy for the global one, but reusing the one from yaml parsing here, I suppose you could reuse the global one too?